models/classes/media/LocalItemSource.php

```diff
@@ -57,69 +58,21 @@ public function getItem()
         return $this->item;
     }
 
+    public function getDirectories(QueryObject $params): array
+    {
+        return $this->searchDirectories($params->getParentLink(), $params->getFilter(), $params->getDepth(),
+            $params->getChildrenLimit());
```

Comment:
To follow PSR-12. The exception (which I do not agree personally) is only about array and lambda parameters

```suggestion
        return $this->searchDirectories(
            $params->getParentLink(), 
            $params->getFilter(), 
            $params->getDepth(),
            $params->getChildrenLimit()
        );
```